sdk/python/packages/flet/src/flet/controls/core/image.py

```diff
@@ -48,6 +48,7 @@ def main(page: ft.Page):
 
     src: OptionalString = None
     src_base64: OptionalString = None
+    src_bytes: Optional[bytes] = None
```

Comment:
We should perhaps think of having just one `src` prop, and have intelligent ifs on dart end.